Lib/pathlib.py

```diff
@@ -1146,21 +1066,40 @@ def readlink(self):
         """
         Return the path to which the symbolic link points.
         """
-        path = self._accessor.readlink(self)
+        if hasattr(os, "readlink"):
+            path = os.readlink(self)
+        else:
+            raise NotImplementedError("os.readlink() not available on this system")
         return self._from_parts((path,))
```

Comment:
Minor nit: I think code becomes simpler if we get rid of that `else`.

```suggestion
        if not hasattr(os, "readlink"):
            raise NotImplementedError("os.readlink() not available on this system")
        path = os.readlink(self)
        return self._from_parts((path,))
```

Reply:
Thanks - I've also applied this change for `symlink()` and `link()`.